contrib/packs/tests/test_action_aliases.py

```diff
@@ -0,0 +1,70 @@
+from st2tests.base import BaseActionAliasTestCase
+
+
+class DeployActionAliasTestCase(BaseActionAliasTestCase):
+    alias_name = 'deploy_pack'
+
+    def test_deploy_alias(self):
+        # Includes packs
+        format_string = self.action_alias_db.formats[0]['representation'][0]
```

Comment:
I'm not too big of a fan of way format strings are referenced, but I couldn't come up with a better approach.

An alternative would be specifying format string in the tests directly (instead of referencing it), but that's kinda defeats the purpose since string would need to be duplicated in two places and things would get out of sync...